src/logic/rsqlService.js

```diff
@@ -0,0 +1,105 @@
+import api from '@molgenis/molgenis-api-client'
+import { transformToRSQL } from '@molgenis/rsql'
+
+function splitText (text) {
+  return text.trim().split(' ')
+}
+
+function queryBuilder (attribute, filters, comparison) {
+  if (!filters.length) return []
+
+  switch (comparison) {
+    case '=in=':
+      return [{ selector: attribute, comparison, arguments: filters }]
+
+    case '=like=':
+      return splitText(filters).map(filter => ({ selector: attribute, comparison: '=like=', arguments: filter }))
+    default:
+      return []
```

Comment:
Shouldn't this throw a warning because it is an unsupported outcome?

Reply:
If it was a library, yes.